tools/c7n_mailer/c7n_mailer/splunk_delivery.py

```diff
@@ -0,0 +1,282 @@
+# Copyright 2019 Capital One Services, LLC
```

Comment:
technically copyright headers on new files should be reflective of the entity that originated the file. i'll update pre-merge.